Editor/BuoyantObjectEditor.cs

```diff
@@ -7,32 +7,38 @@ namespace WaterSystem.Physics
     [CustomEditor(typeof(BuoyantObject))]
     public class BuoyantObjectEditor : Editor
     {
-        private BuoyantObject obj;
+        private BuoyantObject Obj => serializedObject.targetObject as BuoyantObject;
```

Comment:
Does performance matter in this script? There's some overhead added by using a property getter, plus I'm pretty sure it will do the cast each time this is accessed.